docker/katana/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM --platform=$BUILDPLATFORM rust:1.70 as katana-builder
+
+RUN apt-get update && apt-get install -y git
+
+WORKDIR /app
+
+RUN git clone https://github.com/dojoengine/dojo.git
+
+WORKDIR /app/dojo
+
+# add targets
+RUN rustup target add x86_64-unknown-linux-gnu
+RUN rustup target add aarch64-unknown-linux-gnu
+
+# install compilers
+RUN apt-get -y install gcc-x86-64-linux-gnu
+RUN apt-get -y install gcc-aarch64-linux-gnu
+
+# build for both targets
+RUN cargo build --package katana --release --target=x86_64-unknown-linux-gnu --config target.x86_64-unknown-linux-gnu.linker=\"x86_64-linux-gnu-gcc\"
+RUN cargo build --package katana --release --target=aarch64-unknown-linux-gnu --config target.aarch64-unknown-linux-gnu.linker=\"aarch64-linux-gnu-gcc\"
+
+FROM python:3.9.13-slim-bullseye 
+
+HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 \
+    CMD if [ "$(cat /root/.kakarot_deploy_status)" -eq 0 ]; then exit 0; else exit 1; fi
+
+
+COPY --from=katana-builder /app/dojo/target/x86_64-unknown-linux-gnu/release/katana  /app/katana/x86_64-unknown-linux-gnu/release/katana 
+COPY --from=katana-builder /app/dojo/target/aarch64-unknown-linux-gnu/release/katana /app/katana/aarch64-unknown-linux-gnu/release/katana 
+COPY ./docker/katana/scripts /app/scripts
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y build-essential coreutils curl libgmp3-dev
+
+RUN chmod +x scripts/run.sh
+RUN chmod +x scripts/wait_and_deploy.sh
+
+COPY ./docker/katana/cairo/requirements.txt ./requirements.txt
+
+RUN pip3 install -r requirements.txt
```

Comment:
what's the point of this `requirements.txt` file? how has it been generated?

Reply:
I was looking for inspiration for a cairo container, checking if someone has already worked on it before, and found one implementation [here](https://github.com/shramee/starknet-dapp-with-docker/tree/master/python-backend), took a look at the packages being installed there.

Then I double-checked it by looking into the Starknet book [here](https://book.starknet.io/chapter_1/environment_setup.html#starknet_cli_installation), and only kept the ones I saw were necessary, which are the current present ones in the file.

Adding an image here too for more context:
<img width="680" alt="image" src="https://github.com/kkrt-labs/kakarot/assets/41180869/e7341bb9-f445-4491-bd82-e98ee63cf818">

Reply:
if we install the whole project (what we do), we don't need extra lib

Reply:
Gotcha, thanks!

have removed [here](https://github.com/kkrt-labs/kakarot/pull/594/commits/c8c960eba543cf861124467f8ee5fbdb124c0ab5).